docs/src/Callout.doc.js

```diff
@@ -22,168 +47,346 @@ card(
+        `}
+        defaultCode={`
+          <Callout
+            type="info"
+            iconAccessibilityLabel="Info"
+            title="Give $30, get $60 in ads credit"
+            message="Earn $60 of ads credit, and send $30 of ads credit to a friend"
+            primaryAction={{label: "Send invite"}}
+            dismissButton={{
+              accessibilityLabel: 'Dismiss this banner',
+              onDismiss: ()=>{},
+            }}
+          />
+        `}
+      />
+
+      <MainSection.Card
+        cardSize="lg"
+        type="don't"
+        description={`
+        Stack Callouts. In the case that banners must be stacked, Callouts will usually come before Upsells.
+        `}
+        defaultCode={`
+        <Box>
+          <Box marginBottom={4} display="flex" alignItems="center">
+            <Icon accessibilityLabel="" icon="pinterest" color="red" size={32}/>
+            <ButtonGroup>
+              <Button color="transparent" iconEnd="arrow-down" text="Business" inline />
+              <Button color="transparent" iconEnd="arrow-down" text="Create" inline />
+              <Button color="transparent" iconEnd="arrow-down" text="Analytics" inline />
+              <Button color="transparent" iconEnd="arrow-down" text="Ads" inline />
+            </ButtonGroup>
+          </Box>
+          <Divider/>
+          <Box marginTop={8}>
+            <Flex gap={2} direction="column">
+              <Upsell
+                imageData={{
+                  component: <Icon icon="send" accessibilityLabel="Send" color="darkGray" size={32}/>
+                }}
+                title="So close! Finish installing your pinterest tag, get $10 in ads credit"
+                message="Track ads conversion—sales, traffic and more—with the Pinterest tag"
+                primaryAction={{label: "Claim now"}}
+              />
+              <Callout
+                type="info"
+                iconAccessibilityLabel="Info"
+                title="We have not yet detected your tag"
+                message="It may take up to 10 minutes to automatically detect a newly installed tag. If you'd like to manually verify your tag, please click the Verify Tag button."
+                primaryAction={{label: "Verify Tag"}}
+                dismissButton={{
+                  accessibilityLabel: 'Dismiss this banner',
+                  onDismiss: ()=>{},
+                }}
+              />
+            </Flex>
+          </Box>
+        </Box>
+        `}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection name="Accessibility">
+    <MainSection.Subsection
+      title="Labels"
+      description={`
+      \`iconAccessibilityLabel\`, \`dismissButton\`, \`primaryAction\`, and \`secondaryAction\` each require a short, descriptive label for screen readers. These labels should communicate intent or visually describe what any icons might mean, such as “error”, “info” or “warning”. They should also be localized.
```

Comment:
visually describe? what does visually describe mean?

Reply:
Are `These labels` referring to `iconAccessibilityLabel`. Maybe: `The `error`, `info` or `warning` Icon labels should communicate intent or describe what any icons might mean, such as X X X

Not strong about it, just a thought.